subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -1536,6 +1536,14 @@
                 "Method has been removed"
             ]
         },
+        {
+            "type": "org.gradle.kotlin.dsl.ConfigurationDeprecatedExtensionsKt",
+            "member": "Method org.gradle.kotlin.dsl.ConfigurationDeprecatedExtensionsKt.setDescription(org.gradle.api.NamedDomainObjectProvider,java.lang.String)",
+            "acceptation": "Old method is now hidden deprecated",
+            "changes": [
+                "Parameter 1 from null accepting to non-null accepting breaking change"
```

Comment:
It seems the binary compat check is producing a false positive here. Probably due to the way kotlin produces bytcode for deprecated hidden methods. This is fine.

Reply:
Yeah, it has a lot of trouble with Kotlin extensions.